src/style.css

```diff
@@ -1 +1,155 @@
-/* empty source */
+/* #region Variables */
+
+:root {
+  --color_main: #060b35;
+  --color_secondary: #606170;
+  --color_blue: #00acdc;
+}
+
+/* #endregion */
+
+/* #region Normalize */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+/* #endregion */
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-card__img {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.product-card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: var(--color_main);
+  margin-bottom: 4px;
+}
+
+.product-card__code {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: var(--color_secondary);
+  margin-bottom: 16px;
+}
+
+/* #region Product review */
+
+.product-card__review {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+/* #region Product starts */
+
+.stars {
```

Comment:
styles of the other block should go after the styles for the rest of elements
Or into a separate file